hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/client/HoodieFlinkWriteClient.java

```diff
@@ -415,14 +413,13 @@ public void completeTableService(
   }
 
   /**
-   * Upgrade downgrade the Hoodie table.
+   * Try upgrade the Hoodie table.
    *
    * <p>This action should only be executed once for each commit.
    * The modification of the table properties is not thread safe.
    */
-  public void upgradeDowngrade(String instantTime, HoodieTableMetaClient metaClient) {
-    new UpgradeDowngrade(metaClient, config, context, FlinkUpgradeDowngradeHelper.getInstance())
-        .run(HoodieTableVersion.current(), instantTime);
+  public void tryUpgrade(String instantTime, HoodieTableMetaClient metaClient) {
+    tryUpgrade(metaClient, Option.ofNullable(instantTime));
```

Comment:
Hi  @cshuo Is there any context here? Why Flink OperatorCoordinator no need to take care of `downgrade` anymore?

Reply:
The downgrade is not there before even for 0.x release, it's not supported from start.

Reply:
The downgrade is disabled since 1.0.2, hmm~

Reply:
get it ：）